docs/articles/documentation/recipes/best-practices/create-faster-tests.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: docs
+title: Create Faster Tests
+permalink: /documentation/recipes/best-practices/create-faster-tests.html
+redirect_from:
+  - /documentation/recipes/create-faster-tests.html
+---
+# Speed Up Your Tests
+
+The speed of TestCafe tests depends on the loading speed of your web application. The best way to speed up your tests may be to optimize the tested application. Make sure that your application doesn't load large payloads of unused CSS or scripts.
+
+Test in an environment with performance headroom. Lack of resources may increase the time browsers take to initialize, connect to TestCafe and load the tested application. If you have to test in a resource-low environment, [run tests in headless browsers](#run-tests-in-headless-browsers).
+
+For better performance, launch tests in local browsers. Network lag between TestCafe and remote browsers causes delays in test execution.
+
+Remote browsers may take a long time to initialize and connect to TestCafe. Use the  [--browser-init-timeout](../../reference/command-line-interface.md#--browser-init-timeout-ms) CLI option to set a time limit for browsers to connect. Tests fail if this timeout is exceeded - it might be a good time to debug your remote browser connection.
```

Comment:
This is a questionable point. I am unsure if it is viable.

@wentwrong  what do you think?

Reply:
I think it's better to remove this part at all, because it doesn't seem to be related to the disclosed topic of the article.

Reply:
Gone